### ISSUE TITLE
Anti-phospholipase A2 receptor antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7061]
**Preferred term label:**
Anti-phospholipase A2 receptor antibody positivity

**Synonyms**
anti-PLA2R antibody positivity

**Definition (free text, please give PubMed ID)**
PMID: 34277215, PMID: 34169209

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:97560

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added
